homebrew/README.org

```diff
@@ -18,6 +18,20 @@ The run the bleeding edge master build simply pass the =--HEAD= argument.
 brew install --HEAD --formula -vd gerbil-scheme.rb
 #+end_src
 
+When updating/upgrading or hacking it's often needed to clear things.
+
+#+begin_src sh
+  brew unlink gerbil-scheme
+  # Clear the build cache. Helps with subtrees and versions
+  rm -rf $(brew --cache)/gerbil-scheme--git
+#+end_src
+
+And, if worse comes to worse, uninstall and delete the rest.
```

Comment:
worse comes to worst

Reply:
[s/to worse/to Worchestershire/ <-- Capitalization = proper english](https://github.com/mighty-gerbils/gerbil/pull/1100/commits/b56fdbe225fbfeb62d4531b7457af8bec0399a30)

Lowercase e ;) :P